src/encoded/regulome_indexer.py

```diff
@@ -258,9 +254,6 @@ def get_snp_index_mapping(chrom='chr1'):
                 'coordinates': {
                     'type': 'integer_range'
                 },
-                'strand': {
```

Comment:
I guess SNPs don't have a strand?  Or maybe this code is left over from reading it from a BED

Reply:
Right. dbSNP data don't have strand info.